core/extraction/towns_fund.py

```diff
@@ -544,6 +544,7 @@ def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict) -> pd.Dat
     df_funding["Reporting Period"] = [
         x.split(" (£s)__")[1][:3] + x[17:22] if "__" in x else x for x in df_funding["Reporting Period"]
     ]
+    df_funding["Funding Source Name"] = df_funding["Funding Source Name"].str.strip()
```

Comment:
We can also modify the mock submission and expected data for Funding in the unit tests to cover this functionality

Reply:
https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/fd4200d9fafd0154b123a41d63d980c210559b78/tests/extraction_tests/test_round_three_extraction.py#L301

Reply:
I've taken this to mean adding some trailing whitespaces to the mock submission and checking that the expected remains the same. i.e stripped of any spaces

Reply:
yep my bad - that's exactly it